test-runner-backend/terraform/workflow.yaml

```diff
@@ -6,6 +6,7 @@ main:
           - project_id: ${sys.get_env("GOOGLE_CLOUD_PROJECT_ID")}
           - collection: "test-runs"
           - job_id: ${job.jobId}
+          - branch: ${job.branch}
```

Comment:
There are 2 ways for populating branch name :
1. We can pick the branch name using the commit hash in backend, ref [link](https://stackoverflow.com/questions/47086943/how-to-find-the-branch-for-a-commit-with-jgit). If this works then we can pick the branch name and update firestore from backend.
2. The one that is currently done. But this would require us to modify the [PR](https://github.com/graphql-java/graphql-java/pull/2889), where we would have to ask the user for branch name as input and pass it in the payload to workflow. Or if we don't want as input from the user then we need to write some logic to pick the branch name automatically from the Github actions script.

Reply:
Adarsh, thanks for your comment!
Yes, if we are going to leave it as it is now, we need to add a branch input on GitHub actions. What approach do you prefer?